src/Login/Login.tsx

```diff
@@ -1,65 +1,75 @@
-import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
-import './Login.scss';
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./Login.scss";
 
 export interface LoginProps {
-	username: string;
-	setUsername: any;
-	loggedIn: boolean;
-	setLoggedIn: any;
-}
+  username: string;
+  setUsername: Function;
+  loggedIn: boolean;
+  setLoggedIn: Function;
+} //generic ready if we need to safe type any props passed in
 
-const Login: React.SFC<LoginProps> = (props) => {	
-	const verifyUser = () => {
-		props.setLoggedIn(!props.loggedIn);
-	};
+const Login: React.SFC<LoginProps> = (props) => {
+  const verifyUser = () => {
+    props.setUsername("");
+    props.setLoggedIn(!props.loggedIn);
+  };
 
-	const denyUser = (event: any) => {
-		alert('non-alcoholic drinks listing coming soon!')
-	};
+  const denyUser = (event: any) => {
+		return(
+			alert('Non-alcoholic drinks coming soon!')
+		)
+  };
 
-	return (
-		<section className='login-wrapper'>
-			<p className='subtitle'>Learn how to be your own bartender!</p>
-			<form className='login-form'>
-				<p className='login-msg'>You must be 21 or older to use this site. Please enter your name and continue if this applies.</p>
-				<section className="form-input">
-					<input
-						type="name"
-						aria-label="name-input"
-						className="input"
-						placeholder="username"
-						name="username"
-						value={props.username}
-						onChange={(event) => props.setUsername(event.target.value)}
-						required
-					/>
-				</section>
+  const buttonsEnabled = props.username.trim() !== "";
```

Comment:
Functionality that disables both buttons if the input is empty.  
trim() removes any whitespace at beginning or end of input that User may enter.  Whitespace is left if it is surrounded by characters, ie user name

Reply:
Makes sense.